cmake_include/FindMXML.cmake

```diff
@@ -44,8 +44,4 @@ if(NOT HAVE_MXML)
     message(FATAL_ERROR "User requested MXML not found!")
 endif()
 
-if(CMAKE_HOST_UNIX)
-    list(APPEND NAPI_LINK_LIBS mxml)
-else()
-    list(APPEND NAPI_LINK_LIBS mxml mxml1)
-endif()
+list(APPEND NAPI_LINK_LIBS ${MXML_LIBFILES})
```

Comment:
It does work for me, I updated find_module so MXML_LIBFILES it is no longer local variable :-)

The visual studio solution files for MXML call it mxml1.dll, but if you did a MinGW build it might  end up as mxml.dll (haven't checked though) I guess CMAKE_HOST_UNIX doesn't cover MinGW so we would need to use    if (MSVC)    instead?